woocommerce/woocommerce
# Fix PHPCS violations in tests/ using PHPCBF

### Changes proposed in this Pull Request:

This PR fixes all 1533 PHPCS errors that PHPCBF can fix automatically in the tests/ directory. Before this change, there was a total of 3106 PHPCS errors in the WooCommerce repository and now there are 1573 errors.

Since all the modified files are in the tests/ directory, this shouldn't cause any issues.

### How to test the changes in this Pull Request:

1. Skim through the changes to check if they make sense
2. Check that the tests are still passing
